version.properties

```diff
@@ -0,0 +1,3 @@
+#Wed Oct 27 12:41:13 UTC 2021
+VERSION_BUILD_CODE=0
+VERSION_BUILD_STRING=0.00.0
```

Comment:
Why do you need this file?

Reply:
This should not be used. Its only used for applications that need them.  In our case, we already have proper versioning that is coming from gradle.properties. THis is what controls the versioning for the artifacts when they are being deployed

Reply:
@Tom-Ski could you check again please. I have changed it to use talos-version.properties, shall I change it to gradle.properties?